docs/faq.mdx

```diff
@@ -3,8 +3,44 @@ title: Frequently Asked Questions
 description: Find answers to questions about using Pieces for Developers and its many functions.
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { MiniSpacer } from '/src/components/Spacers';
+import CTAButton from "/src/components/CTAButton";
+
 # Frequently Asked Questions
 
+## PiecesOS
+<Tabs defaultValue={"macos"}  values={[{label: 'macOS', value: 'macos'}, {label: 'Windows', value: 'windows'}, {label: 'Linux', value: 'linux'}]}>
+  <TabItem value="macos">
+    <CTAButton
+      href={"/faq/macos"}
+      label={'MacOS'}
+      icon={'/assets/mac_os.png'}
+      iconDark={'/assets/mac_os_dark.png'}
+      type={'secondary'}
```

Comment:
@mason-at-pieces, I'm trying to place the logo near the tab's title. Do you have any tips on how to do this?

Reply:
Don't worry too much about this for now. Just get the content for the first faq from the google doc added and then we can refine things.